tests/CollectorTest.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+// 1. Require in the file you are testing
+require_once 'plant-collect.php';
+
+//2. We get access to PHPUnit
+
+use PHPUnit\Framework\TestCase;
+
+class CollectorTest extends TestCase{
+    //Write your tests here
+    //Test methods must start with 'test'
+    //Make test names descriptive
+    public function testReturnData()
```

Comment:
What kind of test is this? We should include the kind of test (success, failure, malformed) in the function name.

```suggestion
    public function testReturnData_success()
```